account_payment_order/models/account_payment_order.py

```diff
@@ -44,11 +35,19 @@ def domain_journal_id(self):
         readonly=True)
     bank_account_link = fields.Selection(
         related='payment_mode_id.bank_account_link', readonly=True)
+
+    # [bt#5551] From v12
+    allowed_journal_ids = fields.Many2many(
+        comodel_name='account.journal',
+        compute="_compute_allowed_journal_ids",
+        string="Allowed journals",
```

Comment:
If this is intended to be a non-stored computed field, it's okay.

If not, it's a good practice to include the parameters 'relation', 'column1' and 'column2' to identify the table and columns where info will be stored.

Reply:
non-stored field